lib.js

```diff
@@ -19,7 +19,7 @@ function personalBest(person, evt, type='default') {
 
   var matching = person.personalBests.filter((best) => best.eventId === eventId && best.type === type)
   if (matching.length == 0) {
-    return null
+    return new attemptResult.parseString('DNS')
```

Comment:
We can make this switch, but I have no idea what it will affect, so I may switch it back later :) There's special handling for null -- most functions automatically return "null" if one of their arguments is "null", unless the argument is "nullable".

One thing that I can think of is ScrambleSpeedScorer in staff/scorers.js -- this uses your PR to determine whether you should be scrambling an event. I think, with this change, it would strongly prefer people with no PR (since -0.02 a very fast PR!) I would suggest fixing this in AttemptResult.valueOf() -- this has special cases that DNF and 0 are very high numbers, but not DNS.

Reply:
OK good call! Yes that is one part that we did not have to worry about for US Nats :)

Stepping back from the code, I think it would be weird to say that your PR in an event is "DNS". So I like null a bit better from that perspective.

Reply:
I updated the commit fixing this :)